### ISSUE TITLE
Pull in all the wiki stuff?

### ISSUE THREAD

**Issue by vweevers** [Level/awesome#1]
This is.. well, awesome, and could very well replace the [levelup modules](https://github.com/Level/levelup/wiki/Modules) wiki page.

We could then also list compatibility with `levelup@2` and/or `abstract-leveldown@2`.

And then put it on the website! :)

**Comment by MeirionHughes** [comment]
I didn't even know that was there, and I probably wouldn't have made this if I had known. Guess it might get more use if its an open-repo for people to add stuff and correct it. 

Makes sense to go over everything to and list compatibility.

**Comment by vweevers** [comment]
Is there a way to automatically make badges like this?

![abstract-leveldown version](https://img.shields.io/badge/abstract--leveldown-~2.7.1-brightgreen.svg)

**Comment by vweevers** [comment]
Had a look at the `shields.io` service. It's relatively easy to add new badges. For example the npm license badge [queries the npm registry to get the license field from `package.json`](https://github.com/badges/shields/blob/e7f1f57b6b7ce135ad985083f261b66de656ad33/server.js#L1623).

In the same vein, a badge like `https://img.shields.io/npm/dep/leveldown/abstract-leveldown.svg` could request `https://registry.npmjs.org/leveldown/latest` to get the version range of `abstract-leveldown`.

**Comment by vweevers** [comment]
And color it green if up to date, orange if minor outdated, red if major outdated.

**Comment by MeirionHughes** [comment]
Ah okay cool, nice. 👍

**Comment by vweevers** [comment]
I also see a possibility here to explain the ecosystem, from easy to advanced.

I.e. put `level` at the top, saying "the main entry point for beginners - bundles `levelup`, `encoding-down` and `leveldown`". Then list `levelup`, with a description that conveys `levelup` is the glue, without undermining `level`. Then the stores, `encoding-down`, custom encodings, etc. End with the really low-level, half-compatible, gut-wrenchingly monkey-patched stuff.

It'll still be a succinct list. With a little bit of order and assuming each linked module has a good opening paragraph in their readme (`encoding-down` is currently lacking), it can be like a story. Helpful to both beginners and mad scientists.

Can later spruce it up with recipes, talks, popular dependents, what have you.

**Comment by vweevers** [comment]
WIP: https://github.com/Level/awesome/tree/more-awesome

**Comment by ralphtheninja** [comment]
Very nice. Maybe we could "refactor" out all the dependency badges out to the awesome repo?

**Comment by vweevers** [comment]
> Maybe we could "refactor" out all the dependency badges out to the awesome repo?

Hmm not sure. In fact I'm not even sure `awesome` should have *any* badges. Might distract

**Comment by vweevers** [comment]
No, scratch that thought. It's quite valuable here. For example the `level-js` badges clearly indicate it's outdated - and essentially say "you have to know what you're doing" but also "we need some help here :)".

**Comment by MeirionHughes** [comment]
Shame you can't get rid of the titles from each of the badges; they're somewhat redundant if you add the title to the table column.

**Comment by vweevers** [comment]
> Shame you can't get rid of the titles from each of the badges; they're somewhat redundant if you add the title to the table column.

Very true, I looked everywhere but couldn't find an option that does this. It is possible if you add a logo instead of a label, but this currently requires adding a base64-encoded SVG to every badge

**Comment by MeirionHughes** [comment]
close... [![david][01]][02]

super nice work taming the links in the markdown btw. 👍 

[01]: https://img.shields.io/david/level/leveldown.svg?label=&nbsp;
[02]: https://david-dm.org/level/leveldown

**Comment by vweevers** [comment]
#3

**Comment by vweevers** [comment]
To add (after #3):

- [x] [Storage backends](https://github.com/Level/levelup/wiki/Modules#storage-back-ends)
- [ ] Layers
  - [x] `encoding-down`
  - [x] `deferred-leveldown`
  - [x] `cachedown`
  - [ ] `subleveldown` - note: incompatible with `level(up)` v2!
- [ ] [Plugins](https://github.com/Level/levelup/wiki/Modules#plugins)
  -  byteup: "prefer to use bytewise directly"
- [ ] Encodings
  - [ ] bytewise
  - [ ] protobuf
- [ ] [Modules](https://github.com/Level/levelup/wiki/Modules#modules-1)
  - [ ] [Write-streams](https://github.com/Level/levelup/wiki/Modules#write-streams)
  - [ ] Maybe a separate category for indexing & map/reduce
- [ ] [Replication](https://github.com/Level/levelup/wiki/Modules#replication)
- [ ] [Complete databases](https://github.com/Level/levelup/wiki/Modules#complete-databases)
- [ ] [ORMs](https://github.com/Level/levelup/wiki/Modules#orms)
- [ ] [Tools](https://github.com/Level/levelup/wiki/Modules#tools)

**Comment by vweevers** [comment]
@MeirionHughes do you think plugins like your [levelup-async-iterator](https://github.com/MeirionHughes/levelup-async-iterator) should peer-depend on levelup to express compatibility? We can then also derive badges from this

**Comment by vweevers** [comment]
Added a note to the [wiki](https://github.com/Level/levelup/wiki/Modules).
